src/index.js

```diff
@@ -1,80 +1,99 @@
-var supportedOperators = ['contains', 'does not contain', 'is', 'is not', 'is empty', 'is not empty', 'starts with', 'ends with'];
+var supportedOperators = [
```

Comment:
I've gotta admit, I'm surprised it didn't auto change this to `const` - I'm guessing it's because we expose it outside the module and so it could be mutated.

Reply:
It's because we still don't have `no-var` enabled in `eslint-config-mixmax` :(

Reply:
54178ab